typing/outcometree.mli

```diff
@@ -153,6 +153,7 @@ and out_ext_status =
   | Oext_first
   | Oext_next
   | Oext_exception
+  | Oext_effect
```

Comment:
This is a remnant of the syntax for effect declaration and this is not needed anymore.

Reply:
Fixed in https://github.com/ocaml/ocaml/pull/12309/commits/0479cffb3b133c0084683235b231daa02d6bdb6b.